<ui version="4.0" >
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EditDialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2_2" >
     <property name="text" >
      <string>&amp;Phone Number:</string>
     </property>
     <property name="buddy" >
      <cstring>phoneLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="contactLineEdit" />
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLineEdit" name="phoneLineEdit" />
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QLineEdit" name="addressLineEdit" />
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>Ok</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Contact</string>
     </property>
     <property name="buddy" >
      <cstring>contactLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Address</string>
     </property>
     <property name="buddy" >
      <cstring>addressLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>contactLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>EditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>287</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel" >
     <x>143</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>353</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>191</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
